examples/test-unknowns/yaml/Dockerfile

```diff
@@ -1,8 +1,5 @@
 FROM ubuntu AS base
 RUN echo "base"
 
-FROM base AS stage1
-RUN echo "stage1"
```

Comment:
Multi-stage builds probably deserve their own test, I don't think they're doing anything useful for this one.

Reply:
fair, this is a copypaste issue